components/Avatar/avatar.css

```diff
@@ -0,0 +1,89 @@
+@import "/styles.css";
+@import "/docs.css";
+
+.avatar{
+    margin:0.5rem;
+    position:relative;
+    display:inline;
+    height:3rem;
+    width:3rem;
+    border-radius: 50%;
+}
+
```

Comment:
Use consistent blank lines between the two code blocks.

Reply:
Done.